nodes/materials/__init__.py

```diff
@@ -134,8 +134,9 @@ def poll(cls, context):
         NodeItem("LuxCoreNodeTexConstfloat3", label="Constant Color"),
         NodeItem("LuxCoreNodeTexPointiness", label="Pointiness"),
         NodeItem("LuxCoreNodeTexUV", label="UV Test"),
+        NodeItem("LuxCoreNodeIORPreset", label="IOR Preset"),
```

Comment:
You should also add it to the utils category of texture node trees in nodes/textures/__init__.py.